src/server/scripts/Northrend/VaultOfArchavon/boss_emalon.cpp

```diff
@@ -173,6 +173,7 @@ class boss_emalon : public CreatureScript
                     case EVENT_BERSERK:
                         me->CastSpell(me, SPELL_BERSERK, true);
                         Talk(EMOTE_BERSERK);
+                        events.PopEvent();
```

Comment:
this might no longer be needed after https://github.com/azerothcore/azerothcore-wotlk/pull/3426

Reply:
@FrancescoBorzi Later night I will check if, with that PR changes, the same issue here takes place.